core/tests/protractor_utils/CreatorDashboardPage.js

```diff
@@ -91,6 +93,14 @@ var CreatorDashboardPage = function() {
     waitFor.pageToFullyLoad();
   };
 
+  this.clickCreateExplorationButtonOnMobile = function() {
```

Comment:
Can you update the name to have something like "clickCreateButtonInNavbarOnDesktop" or something like that? I was confused too.

Reply:
Just a doubt here -- this isn't the navbar, this is the "Create Exploration" button. So I am naming it to `clickCreateExplorationButton` since this is same for both desktop and mobile. Does this look right?

Reply:
Yep seems fine to me. If the element exists in both desktop and mobile then there should be just one function for it and (ideally) it shouldn't branch.

Reply:
Done, thanks!